tests/common.go

```diff
@@ -52,28 +56,30 @@ const (
 	provisionerFlag                      = "provisioner"
 	storageNodesPerAZFlag                = "max-storage-nodes-per-az"
 	configMapFlag                        = "config-map"
+	scaleStorageFactorFlag               = "scale-storage-factor"
 )
 
 const (
-	defaultScheduler                      = "k8s"
-	defaultNodeDriver                     = "ssh"
-	defaultStorageDriver                  = "pxd"
-	defaultLogLocation                    = "/mnt/torpedo_support_dir"
-	defaultLogLevel                       = "debug"
-	defaultAppScaleFactor                 = 1
-	defaultMinRunTimeMins                 = 0
-	defaultChaosLevel                     = 5
-	defaultStorageUpgradeEndpointURL      = "https://install.portworx.com/upgrade"
-	defaultStorageUpgradeEndpointVersion  = "2.1.1"
-	defaultStorageProvisioner             = "portworx"
-	defaultStorageNodesPerAZ              = 2
-	defaultAutoStorageNodeRecoveryTimeout = 30 * time.Minute
+	defaultScheduler                     = "k8s"
+	defaultNodeDriver                    = "ssh"
+	defaultStorageDriver                 = "pxd"
+	defaultLogLocation                   = "/mnt/torpedo_support_dir"
+	defaultLogLevel                      = "debug"
+	defaultAppScaleFactor                = 1
+	defaultMinRunTimeMins                = 0
+	defaultChaosLevel                    = 5
+	defaultStorageUpgradeEndpointURL     = "https://install.portworx.com/upgrade"
+	defaultStorageUpgradeEndpointVersion = "2.1.1"
+	defaultStorageProvisioner            = "portworx"
+	defaultStorageNodesPerAZ             = 2
+	storagePoolResizeAnnotationKey       = "torpedo.io/autopilot-storage-pool-resize-enabled"
```

Comment:
you have the `torpedo.io/autopilot-storage-pool-resize-enabled` key declared at 2 places

```
drivers/scheduler/k8s/k8s.go:   autopilotStoragePoolResizeEnabledAnnotationKey = "torpedo.io/autopilot-storage-pool-resize-enabled"
tests/common.go:        storagePoolResizeAnnotationKey       = "torpedo.io/autopilot-storage-pool-resize-enabled"
```

Reply:
Actually `storagePoolResizeAnnotationKey` doesn't look like it's being referenced anywhere. This can be removed.